app/Extensions/BackupAdapter/Schemas/S3BackupSchema.php

```
@@ -0,0 +1,207 @@
+
+    public function deleteBackup(Backup $backup): void
+    {
+        $client = $this->createClient($backup->backupHost);
+
+        $client->deleteObject([
+            'Bucket' => $backup->backupHost->configuration['bucket'],
+            'Key' => "{$backup->server->uuid}/$backup->uuid.tar.gz",
+        ]);
+    }
+
+    public function getDownloadLink(Backup $backup, User $user): string
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$user'. (undefined)

(UnusedFormalParameter)